idm/meta/dao/sql/ns-sql.go

```
@@ -96,7 +95,12 @@ func (u *MetaNamespace) From(res *idm.UserMetaNamespace) *MetaNamespace {
 	return u
 }
 
-func (u *MetaNamespace) FromExisting(res *idm.UserMetaNamespace) *MetaNamespace {
+func (u *MetaNamespace) FromExisting(res *idm.UserMetaNamespace) (*MetaNamespace, error) {
+	jssc := json_schema.ValidateSchemaFromPbStruct(res.JsonSchema)
```

This validation could be moved to another layer.

---

not a big issue to me, better late than ignore errors ;-)